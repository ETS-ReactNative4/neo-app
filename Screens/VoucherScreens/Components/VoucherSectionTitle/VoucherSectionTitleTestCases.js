import React from "react";

const VoucherSectionTitleTestCases = [];

export default VoucherSectionTitleTestCases;
