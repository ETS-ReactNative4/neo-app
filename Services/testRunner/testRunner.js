import renderer from "react-test-renderer";

const testRunner = testCase => {
  test(testCase.title, () => {
    const tree = renderer.create(testCase.Component).toJSON();
    expect(tree).toMatchSnapshot();
  });
};

export default testRunner;
