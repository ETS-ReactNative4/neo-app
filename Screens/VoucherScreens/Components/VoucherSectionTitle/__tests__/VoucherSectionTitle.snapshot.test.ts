import VoucherSectionTitleTestCases from "../VoucherSectionTitleTestCases";
import testRunner from "../../../../../Services/testRunner/testRunner";

VoucherSectionTitleTestCases.map(testRunner);
