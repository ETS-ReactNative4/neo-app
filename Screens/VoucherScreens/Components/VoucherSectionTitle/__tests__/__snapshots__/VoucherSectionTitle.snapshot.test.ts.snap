// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Voucher Section Title - without icon 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "flex-start",
        "flexDirection": "row",
        "justifyContent": "flex-start",
        "padding": 24,
      },
      Object {},
    ]
  }
>
  <View
    style={
      Array [
        Object {},
        null,
      ]
    }
  >
    <Text
      style={
        Object {
          "color": "rgba(44, 47, 66, 1)",
          "fontFamily": "SourceSansPro-SemiBold",
          "fontSize": 14,
          "lineHeight": 14,
        }
      }
    >
      Hotel Room 1
    </Text>
  </View>
</View>
`;

exports[`Voucher Section Title 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "flex-start",
        "flexDirection": "row",
        "justifyContent": "flex-start",
        "padding": 24,
      },
      Object {},
    ]
  }
>
  <View
    style={Object {}}
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "rgba(119, 119, 119, 1)",
            "fontSize": 14,
          },
          undefined,
          Object {
            "fontFamily": "Voyager",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      Ó§Å
    </Text>
  </View>
  <View
    style={
      Array [
        Object {},
        Object {
          "marginLeft": 8,
          "marginTop": 2,
        },
      ]
    }
  >
    <Text
      style={
        Object {
          "color": "rgba(44, 47, 66, 1)",
          "fontFamily": "SourceSansPro-SemiBold",
          "fontSize": 14,
          "lineHeight": 14,
        }
      }
    >
      Hotel Room 1
    </Text>
  </View>
</View>
`;
